cylc/flow/task_pool.py

```diff
@@ -1300,63 +1300,6 @@ def ping_task(self, id_, exists_only=False):
         else:
             return False, "task not found"
 
-    def get_task_requisites(self, items, list_prereqs=False):
-        """Return task prerequisites.
-
-        Result in a dict of a dict:
-        {
-            "task_id": {
-                "meta": {key: value, ...},
-                "prerequisites": {key: value, ...},
-                "outputs": {key: value, ...},
-                "extras": {key: value, ...},
-            },
-            ...
-        }
-        """
-        itasks, bad_items = self.filter_task_proxies(items)
-        results = {}
-        now = time()
-        for itask in itasks:
-            if list_prereqs:
-                results[itask.identity] = {
-                    'prerequisites': itask.state.prerequisites_dump(
-                        list_prereqs=True)}
-                continue
-
-            extras = {}
-            if itask.tdef.clocktrigger_offset is not None:
-                extras['Clock trigger time reached'] = (
-                    itask.is_waiting_clock_done(now))
-                extras['Triggers at'] = get_time_string_from_unix_time(
```

Comment:
Now the `get_time_string_from_unix_time` can be removed from the imports in this file, on line 58.